setup.py

```diff
@@ -82,6 +82,10 @@ def main():
         packages=["returnn"],
         include_package_data=True,
         package_data={"returnn": package_data},  # filtered via MANIFEST.in
+        install_requires=[
+            "numpy",
+            "h5py",
```

Comment:
Note: h5py should be an optional dependency, not necessarily required. (If it is, consider this a bug.)
```suggestion
```

Reply:
In that case you should want to set it as an extra.

I'm gonna hit "accept" on the suggestion and leave it with you to fix that up.